packages/common/src/project/readers/github-reader.ts

```diff
@@ -17,25 +17,23 @@ export class GithubReader implements Reader {
     });
   }
 
+  get root(): undefined {
+    return undefined;
+  }
+
   async getPkg(): Promise<IPackageJson | undefined> {
-    return this.getFile('package.json');
+    return this.getFile('package.json') as Promise<unknown> as Promise<IPackageJson>;
```

Comment:
What if `package.json` doesn't exist?
```suggestion
    return this.getFile('package.json') as Promise<unknown> as Promise<IPackageJson | undefined>;
```